features/support/xmlrpc_system.rb

```diff
@@ -40,4 +40,9 @@ def createSystemRecord(sys_name, ks_label, ip, mac)
     netdevs = [netdev]
     @connection.call('system.createSystemRecord', @sid, sys_name, ks_label, '', '', netdevs)
   end
+
+  # Bootstrap a system (regular salt minion)
+  def bootstrapSystem(host, ssh_user, ssh_password, activation_key, salt_ssh)
+    @connection.call('system.bootstrap', @sid, host, 22, ssh_user, ssh_password, activation_key, salt_ssh)
```

Comment:
Actually, I'd go in a different direction. Parametrizing this also makes rubocop complain and brings additional complexity that is not needed. In our test scenarios, we only parametrize the `host` and `saltSSH` parameters. When the tests grow, I'd be more than happy to extend this function as well, but when tests are like that, I'd keep it simple.